spec/browser-info.spec.mjs

```diff
@@ -0,0 +1,16 @@
+import * as browser from "../jspsych-uil-browser.js";
+
+describe('browser info', () => {
+    let info = browser.getBrowserInfo();
+
+    it('should not be empty', () => {
+        expect(Object.keys(info).length).toBeGreaterThan(0);
+    });
+
+    it('should be correct for headless chrome', () => {
+        expect(info.browser).toContain('Chrome Headless');
+        expect(info.isMobile).toBeFalse();
+        expect(info.isTablet).toBeFalse();
+        expect(info.isTouchCapable).toBeFalse();
+    });
```

Comment:
This one might be a bit of a pain. I'd like to run the test with
```bash
npx jasmine-browser-runner
```
and then navigate to localhost:8888 in firefox or chrome
This is great for the CI i suppose, but perhaps less nice for running the test with a browser and its development tools.

Reply:
I think the browser name part can go, it's not super important